### ISSUE TITLE
Can't list zips containing more 48 files

### ISSUE THREAD

**Issue by xperiments** [bitbank2/unzipLIB#16]
Can't list zips containing more 48 files

While other bigger zip in size works, locateFile fails if there is more > 48 files in zip

Thxxxx

**Comment by xperiments** [comment]
[demo.gcode.zip](https://github.com/user-attachments/files/17683941/demo.gcode.zip)

**Comment by SourceCodeDeleted** [comment]
will it extract them?

**Comment by bitbank2** [comment]
> Can't list zips containing more 48 files
> 
> While other bigger zip in size works, locateFile fails if there is more > 48 files in zip
> 
> Thxxxx

My code is just a wrapper over the original unzip code. Limitations like this are not my doing. Can you provide a fix?

**Comment by SourceCodeDeleted** [comment]
No, I just went with a different lib which literally produced two files.
